internal/app/cli.go

```diff
@@ -35,7 +58,8 @@ func (i *stringArray) Set(value string) error {
 
 type cli struct {
 	debug                 bool
-	files                 stringArray
+	files                 fileOptionArray
```

Comment:
Does this affect the existing functionality of passing multiple state files using the `-f` flag? Has this been tested with your branch?

Reply:
I've checked that when adding my changes, worked as it was before